File: src/JavaScript/CreationalPatterns/AbstractFactory/EraStyles/products/victorianClothing.js

const EraObject = require("../abstraction/eraObject");
const GlobalConstants = require("../constants/globalConstants");

class VictorianClothing extends EraObject {
    showDetails() {
        console.log(GlobalConstants.ClothingMessage.replace("{0}", "Victorian Era"));
    }
}

module.exports = VictorianClothing;

File: src/C#/CreationalPatterns/AbstractFactory/EraStyles/Products/VictorianClothing.cs
using EraStyles.Abstraction;
using EraStyles.Constants;
using System;
using System.Collections.Generic;
using System.Text;

namespace EraStyles.Products
{
    public class VictorianClothing : EraObject
    {
        public override void ShowDetails()
        {
            Console.WriteLine(string.Format(GlobalConstants.ClothingMessage, "Victorian Era"));
        }
    }
}
